docs/user-guide/index.rst

```diff
@@ -0,0 +1,33 @@
+.. _user-guide:
+
+User Guide
+==========
+
+.. toctree::
+    :maxdepth: 1
+
+    arrays
+    groups
+    attributes
+    storage
+    config
+
+.. Coming soon
+    installation
+    v3_migration
+
+Advanced Topics
+---------------
+
+.. toctree::
+    :maxdepth: 1
+
+    performance
+    consolidated_metadata
+    whatsnew_v3
```

Comment:
It doesn't feel right for a what's new page to be under Advanced topics - perhaps under a new "Zarr v3" section instead, which can also have the migration guide and TODOs pages?